synapse/storage/databases/main/roommember.py

```diff
@@ -152,13 +152,18 @@ async def get_users_in_room(self, room_id: str) -> List[str]:
         `get_current_hosts_in_room()` and so we can re-use the cache but it's
         not horrible to have here either.
 
-        Uses `m.room.member`s in the room state at the current forward extremities to
-        determine which users are in the room.
-
-        Will return inaccurate results for rooms with partial state, since the state for
-        the forward extremities of those rooms will exclude most members. We may also
-        calculate room state incorrectly for such rooms and believe that a member is or
-        is not in the room when the opposite is true.
+        Uses `m.room.member`s in the room state at the current forward
+        extremities to determine which users are in the room.
+
+        Will return inaccurate results for rooms with partial state, since the
+        state for the forward extremities of those rooms will exclude most
+        members. We may also calculate room state incorrectly for such rooms and
+        believe that a member is or is not in the room when the opposite is
+        true.
+
+        Note: If you only care about users in the room local to the homeserver,
+        use `get_local_users_in_room(...)` instead which will be more
+        performant.
```

Comment:
Only change is this last paragraph to point prospective users of `get_users_in_room` to consider using `get_local_users_in_room` instead if it fits their use case.

The rest of the changes is auto-wrapping.

Reply:
I think we don't need to rewrap these lines? We've set up our linters to 88 columns, instead of 80.

Reply:
Ahh, thanks @squahtx, adjusted the wrapping in my text editor 🙂